docs: re-update assets for specs

seolaoh opened the pull request:
Re-updated some assets to properly represent Kanvas' architecture.

chokobole commented:
Could you tell me why you changed the assets?

If you intend to use abstract terms in the overview, it would be good to have consistent terminology throughout the document, as some parts refer to 'Execution Engine' while others use 'Execution Engine(EE).'

Is it correct that you juste removed spaces in the last picture?

seolaoh commented on specs/assets/architecture.svg:
unify component terms to Batcher, Rollup Node, Validator, Execution Engine

seolaoh commented on specs/assets/components.svg:
add spacing between arrow lines

seolaoh commented on specs/assets/propagation.svg:
unify component terms to Execution Engine, KanvasPortal (not Portal)

seolaoh commented on specs/assets/user-withdrawing-to-l1.svg:
just remove spaces in L2OutputOracle and BatchInBox

seolaoh commented:
> Could you tell me why you changed the assets?
> 
> If you intend to use abstract terms in the overview, it would be good to have consistent terminology throughout the document, as some parts refer to 'Execution Engine' while others use 'Execution Engine(EE).'
> 
> Is it correct that you juste removed spaces in the last picture?

I added some comments to each picture.

> Could you tell me why you changed the assets?
> 
> If you intend to use abstract terms in the overview, it would be good to have consistent terminology throughout the document, as some parts refer to 'Execution Engine' while others use 'Execution Engine(EE).'
> 
> Is it correct that you juste removed spaces in the last picture?

@chokobole I added some comments to each picture.

chokobole reviewed (approved):
lgtm
